template/.github/pull_request_template.md

```
@@ -1,21 +1,21 @@
- ## Link to Issue or Message thread
+## Link to Issue or Message thread
```

markdownlint-cli2 (0.21.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)